tee-worker/litentry/core/assertion-build/src/a3.rs

```diff
@@ -24,8 +24,8 @@ use crate::*;
 use lc_data_providers::{discord_litentry::DiscordLitentryClient, vec_to_string};
 
 const VC_A3_SUBJECT_DESCRIPTION: &str =
-	"The user has commented in a specific Discord channel with a specific role";
-const VC_A3_SUBJECT_TYPE: &str = "Discord Member Verification";
+	"You have commented in Litentry Discord #🪂id-hubber channel. Channel link: https://discord.com/channels/807161594245152800/1093886939746291882";
```

Comment:
Is this emoji intended? 😲

Reply:
Yes, after confirming with Kayla again, this was intentionally added.